app/views/persons/_mobile.html.erb

```diff
@@ -3,7 +3,7 @@
     <% if @person.image.attached?  %>
       <%= image_tag @person.show_image, class: "decorative" %>
     <% end %>
-    <h1 class="person-name"><%= @person.first_name %> <%= @person.last_name %></h1>
+    <h1 class="person-name"><%= title @person.first_name + " " + @person.last_name %></h1>
```

Comment:
Please use string interpolation instead of concatenation
`<%= title "#{@person.first_name}  #{@person.last_name}" %>`

It's more idiomatic ruby, plus it is safer against unexpected bad data like `@person.first_name` being nil.

Reply:
got it